File: templates/dart-minimal/files/test/smoke_test.dart
import 'package:test/test.dart';
import 'package:{{ app_name }}/{{ app_name }}.dart';

void main() {
  group('Smoke', () {
    test('package loads and exposes version', () {
      expect(version, equals('0.0.1'));
    });
  });
}

File: src/loom/templates/typescript-minimal/files/tests/smoke.test.ts
import { describe, test, expect } from 'vitest';
import { version, greeting } from '../src/index.js';

describe('Smoke', () => {
  test('package exposes version', () => {
    expect(version).toBe('0.0.1');
  });

  test('greeting interpolates name', () => {
    expect(greeting('world')).toBe('Hello, world!');
  });
});
